python/perf-kernels/flash-attention.py

```diff
@@ -216,7 +266,7 @@ def _attn_fwd(
         elif bias_type == "matrix":
             bias_ptr = tl.make_block_ptr(
                 base=bias + ((off_hz % H) * stride_bh),
-                shape=(N_CTX, N_CTX),
+                shape=(seqlen_k, seqlen_k),
```

Comment:
should this be seqlen_q x seqlen_k since the intermediate tensor is the produc of Q x K?